db/migrate/20161008201253_create_urls.rb

```diff
@@ -0,0 +1,9 @@
+class CreateUrls < ActiveRecord::Migration
+  def change
+    create_table :urls do |t|
+      t.string :url
```

Comment:
i encourage you not to name an attribute of a table the same name as the table itself. this becomes very confusing. we don't want to ever call `url.url`. it's not readable at all. here - maybe the table is `urls` and the attribute is `address`?